data-raw/dummy_anchored.R

```diff
@@ -73,8 +73,11 @@ agd <- process_agd(agd)
 adsl_twt <- dummize_ipd(adsl_twt, dummize_cols = c("SEX"), dummize_ref_level = c("Female"))
 centered_ipd_twt <- center_ipd(ipd = adsl_twt, agd = agd)
 
+centered_colnames <- paste0(c("AGE", "AGE_MEDIAN", "AGE_SQUARED", "SEX_MALE", "ECOG0", "SMOKE"), "_CENTERED")
+weighted_twt <- estimate_weights(data = centered_ipd_twt, centered_colnames = centered_colnames, n_boot_iteration = 1000)
```

Comment:
Can you see how long it takes to run `maic_anchored()` with 1000 samples? If it's more than a few seconds we should reduce N

Reply:
it takes 8 sec for 1000 samples, 1 sec for 100 samples. Which?

Reply:
Better go for 100 then. We will have trouble on CRAN otherwise

Reply:
okay will run with 100!